pkg/runtime/service_controller.go

```diff
@@ -133,13 +133,15 @@ func (c *serviceController) BindControllerManager(mgr ctrlrt.Manager, cfg ackcfg
 	c.metaLock.Lock()
 	defer c.metaLock.Unlock()
 
-	clusterConfig := mgr.GetConfig()
-	clientset, err := kubernetes.NewForConfig(clusterConfig)
-	if err != nil {
-		return err
+	cache := ackrtcache.New(c.log)
+	if cfg.WatchNamespace != "" {
```

Comment:
```suggestion
	if cfg.WatchNamespace == "" {
```
Aaaaaah... :facepalm:

Reply:
This is why you still see the error @ryansteakley

Reply:
Fixed. Can you try again @ryansteakley?

Reply:
testing it out now

Reply:
Worked for getting rid of the errors related to v1.Namespace, but now I'm seeing that although the resources are being created on sagemaker and I can also successfully delete them, none of this is reflected in kubectl get or kubectl describe as they don't seem to be updating.

Reply:
@RedbackThomson please see above... is this anything to do with the GenerationPredicate bug you told me about?

Reply:
I believe the `GenerationPredicate` bug has made this visible. On first reconciliation we were getting an error, it was updating the spec but not the status. On the second reconciliation, it updated the status.